packages/pwa/app/pages/account/order-detail.test.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React, {useEffect} from 'react'
+import {screen} from '@testing-library/react'
+import {renderWithProviders} from '../../utils/test-utils'
+import useCustomer from '../../commerce-api/hooks/useCustomer'
+import {AccountOrdersProvider} from './util/order-context'
+import OrderDetail from './order-detail'
+import {keysToCamel} from '../../commerce-api/utils'
+import {
+    mockedRegisteredCustomer,
+    mockOrderHistory,
+    mockOrderProducts,
+    ocapiOrderResponse
+} from '../../commerce-api/mock-data'
+
+const mockOrder = keysToCamel({
+    basket_id: 'testorderbasket',
+    ...ocapiOrderResponse
+})
+
+jest.setTimeout(30000)
+
+const MockedOrderDetails = () => {
+    const customer = useCustomer()
+    useEffect(() => {
+        customer.login('test@test.com', 'password')
+    }, [])
+
+    //Only render orders after login is complete
+    if (!customer.customerId) {
+        return null
+    }
+    return (
+        <div>
+            <div>{customer.customerId}</div>
+            <AccountOrdersProvider>
+                <OrderDetail />
+            </AccountOrdersProvider>
+        </div>
+    )
+}
```

Comment:
I believe this is a bad example of a unit test that @kevinxh is referring to in the example. 
Why are we not mocking the `useCustomer` hook here?